requirements.txt

```diff
@@ -7,6 +7,7 @@ jsonschema==2.5.1
 # regulations-site
 # django already covered
 requests==2.8.1
+requests-toolbelt==0.6.0
```

Comment:
I think you said this would be needed in the future -- just confirming that this was intentional?